lms/djangoapps/ccx/utils.py

```diff
@@ -127,6 +127,31 @@ def get_ccx_for_coach(course, coach):
     return None
 
 
+def get_ccx_by_ccx_id(course, coach, ccx_id):
+    """
+    Looks to see if user is coach of a CCX for this course.  Returns the CCX or
+    None.
```

Comment:
Please word the docstring as, "Finds a CCX that ....".  This string says "a CCX", but you are given the ccx_id, so there's a very particular CCX in mind, or am I misunderstanding?